Build errors starting 2025-02-25

Builds of this action started failing on 2025-02-25. A sample of the tail of the build logs is included below.

```log
...
2025-02-25T22:35:14.6496158Z #13 6.676 added 251 packages in 5s
2025-02-25T22:35:14.7999884Z #13 6.676 
2025-02-25T22:35:14.8000368Z #13 6.676 40 packages are looking for funding
2025-02-25T22:35:14.8000811Z #13 6.676   run `npm fund` for details
2025-02-25T22:35:15.9491632Z #13 7.976 Collecting func_timeout
2025-02-25T22:35:16.1666738Z #13 8.021   Downloading func_timeout-4.3.5.tar.gz (44 kB)
2025-02-25T22:35:16.1667656Z #13 8.032      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 44.3/44.3 kB 4.2 MB/s eta 0:00:00
2025-02-25T22:35:16.1668421Z #13 8.043   Preparing metadata (setup.py): started
2025-02-25T22:35:16.3233931Z #13 8.350   Preparing metadata (setup.py): finished with status 'done'
2025-02-25T22:35:16.4256883Z #13 8.405 Collecting networkx
2025-02-25T22:35:16.4257535Z #13 8.414   Downloading networkx-3.1-py3-none-any.whl (2.1 MB)
2025-02-25T22:35:16.5551036Z #13 8.452      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.1/2.1 MB 57.7 MB/s eta 0:00:00
2025-02-25T22:35:16.5551640Z #13 8.483 Collecting sty==1.0.0rc0
2025-02-25T22:35:16.5552150Z #13 8.492   Downloading sty-1.0.0rc0-py3-none-any.whl (11 kB)
2025-02-25T22:35:16.5552673Z #13 8.582 Collecting tqdm
2025-02-25T22:35:16.7286521Z #13 8.609   Downloading tqdm-4.67.1-py3-none-any.whl (78 kB)
2025-02-25T22:35:16.7287695Z #13 8.614      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.5/78.5 kB 25.6 MB/s eta 0:00:00
2025-02-25T22:35:16.7309073Z #13 8.632 ERROR: Ignored the following versions that require a different python version: 3.2 Requires-Python >=3.9; 3.2.1 Requires-Python >=3.9; 3.2rc0 Requires-Python >=3.9; 3.3 Requires-Python >=3.10; 3.3rc0 Requires-Python >=3.10; 3.4 Requires-Python >=3.10; 3.4.1 Requires-Python >=3.10; 3.4.2 Requires-Python >=3.10; 3.4rc0 Requires-Python >=3.10
2025-02-25T22:35:16.7310616Z #13 8.632 ERROR: Could not find a version that satisfies the requirement pysimplegui (from versions: 5.0.0, 5.0.2, 5.0.3, 5.0.4, 5.0.5, 5.0.6, 5.0.7, 5.0.8)
2025-02-25T22:35:16.7311250Z #13 8.632 ERROR: No matching distribution found for pysimplegui
2025-02-25T22:35:16.7311544Z #13 8.755 
2025-02-25T22:35:16.7311796Z #13 8.755 [notice] A new release of pip is available: 23.0.1 -> 25.0.1
2025-02-25T22:35:16.7312157Z #13 8.755 [notice] To update, run: pip install --upgrade pip
2025-02-25T22:35:16.9422248Z #13 ERROR: process "/bin/sh -c ./install.sh" did not complete successfully: exit code: 1
2025-02-25T22:35:16.9510680Z ------
2025-02-25T22:35:16.9511283Z  > [9/9] RUN ./install.sh:
2025-02-25T22:35:16.9511997Z 8.492   Downloading sty-1.0.0rc0-py3-none-any.whl (11 kB)
2025-02-25T22:35:16.9512628Z 8.582 Collecting tqdm
2025-02-25T22:35:16.9513141Z 8.609   Downloading tqdm-4.67.1-py3-none-any.whl (78 kB)
2025-02-25T22:35:16.9513867Z 8.614      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.5/78.5 kB 25.6 MB/s eta 0:00:00
2025-02-25T22:35:16.9516818Z 8.632 ERROR: Ignored the following versions that require a different python version: 3.2 Requires-Python >=3.9; 3.2.1 Requires-Python >=3.9; 3.2rc0 Requires-Python >=3.9; 3.3 Requires-Python >=3.10; 3.3rc0 Requires-Python >=3.10; 3.4 Requires-Python >=3.10; 3.4.1 Requires-Python >=3.10; 3.4.2 Requires-Python >=3.10; 3.4rc0 Requires-Python >=3.10
2025-02-25T22:35:16.9520299Z 8.632 ERROR: Could not find a version that satisfies the requirement pysimplegui (from versions: 5.0.0, 5.0.2, 5.0.3, 5.0.4, 5.0.5, 5.0.6, 5.0.7, 5.0.8)
2025-02-25T22:35:16.9521395Z 8.632 ERROR: No matching distribution found for pysimplegui
2025-02-25T22:35:16.9521899Z 8.755 
2025-02-25T22:35:16.9522314Z 8.755 [notice] A new release of pip is available: 23.0.1 -> 25.0.1
2025-02-25T22:35:16.9522925Z 8.755 [notice] To update, run: pip install --upgrade pip
2025-02-25T22:35:16.9523404Z ------
2025-02-25T22:35:16.9523705Z Dockerfile:19
2025-02-25T22:35:16.9524000Z --------------------
2025-02-25T22:35:16.9524330Z   17 |     RUN apt-get update \
2025-02-25T22:35:16.9525000Z   18 |     	&& apt-get install npm -y
2025-02-25T22:35:16.9525396Z   19 | >>> RUN ./install.sh
2025-02-25T22:35:16.9525722Z   20 |     
2025-02-25T22:35:16.9526191Z --------------------
2025-02-25T22:35:16.9526811Z ERROR: failed to solve: process "/bin/sh -c ./install.sh" did not complete successfully: exit code: 1
2025-02-25T22:35:16.9557628Z ##[endgroup]
2025-02-25T22:35:16.9644093Z ##[error]Docker build failed with exit code 1
2025-02-25T22:35:16.9889533Z ##[group]Run step-security/harden-runner@0080882f6c36860b6ba35c610c98ce87d4e2f26f
2025-02-25T22:35:16.9890217Z with:
2025-02-25T22:35:16.9890402Z   disable-sudo: true
2025-02-25T22:35:16.9900706Z   egress-policy: block
2025-02-25T22:35:16.9901002Z   allowed-endpoints: github.com:443

2025-02-25T22:35:16.9901401Z   token: ***
2025-02-25T22:35:16.9901597Z   disable-telemetry: false
2025-02-25T22:35:16.9901830Z   disable-file-monitoring: false
2025-02-25T22:35:16.9902450Z ##[endgroup]
2025-02-25T22:35:17.2570225Z [harden-runner] pre-step
2025-02-25T22:35:17.2577865Z [!] Current Configuration: 
2025-02-25T22:35:17.2579647Z {"repo":"ericcornelissen/eslint-plugin-top","run_id":"13532288617","correlation_id":"1902866b-96f6-411a-a3a7-3e6e8cd9c727","working_directory":"/home/runner/work/eslint-plugin-top/eslint-plugin-top","api_url":"https://agent.api.stepsecurity.io/v1","allowed_endpoints":"github.com:443","egress_policy":"block","disable_telemetry":false,"disable_sudo":true,"disable_file_monitoring":false,"private":false,"is_github_hosted":true,"is_debug":false,"one_time_key":""}
2025-02-25T22:35:17.2581106Z 
2025-02-25T22:35:17.2782932Z [command]/usr/bin/tar --posix -cf cache.tzst --exclude cache.tzst -P -C /home/runner/work/eslint-plugin-top/eslint-plugin-top --files-from manifest.txt --use-compress-program zstdmt
2025-02-25T22:35:17.4369997Z Failed to save: Unable to reserve cache with key harden-runner-cacheKey, another job may be creating this cache. More details: Cache already exists. Scope: refs/pull/1383/merge, Key: harden-runner-cacheKey, Version: 60107ed97f01a753b09a5130019225dcf6640b6eecd3ec268cc1d165dd5ff39c
2025-02-25T22:35:17.4377237Z -1
2025-02-25T22:35:17.4424353Z cacheFilePath /home/runner/work/_actions/step-security/harden-runner/0080882f6c36860b6ba35c610c98ce87d4e2f26f/dist/pre/cache.txt
2025-02-25T22:35:17.4589489Z Adding cacheHost: kv4gacprodeus2file3.blob.core.windows.net:443 to allowed-endpoints
2025-02-25T22:35:17.4590849Z [32mView security insights and recommended policy at:[0m
2025-02-25T22:35:17.4591773Z https://app.stepsecurity.io/github/ericcornelissen/eslint-plugin-top/actions/runs/13532288617
2025-02-25T22:35:17.4592585Z RUNNER_NAME: GitHub Actions 1
2025-02-25T22:35:18.2920658Z Runner IP Address: 20.161.77.251
2025-02-25T22:35:18.2921404Z Step Security Job Correlation ID: 1902866b-96f6-411a-a3a7-3e6e8cd9c727
2025-02-25T22:35:18.3115358Z [!] Checking TLS_STATUS: ericcornelissen
2025-02-25T22:35:18.4150347Z [!] TLS_NOT_ENABLED: ericcornelissen
2025-02-25T22:35:18.6098760Z [command]/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/939fea1d-6834-45ab-9b06-da485d76e8d8 -f /home/runner/work/_temp/f8198aed-5dc6-467d-aec4-021de82a10a7
2025-02-25T22:35:21.0096953Z Initialized
2025-02-25T22:35:21.0262782Z Post job cleanup.
2025-02-25T22:35:21.0654798Z [harden-runner] post-step
2025-02-25T22:35:22.0693206Z log:
2025-02-25T22:35:22.0693840Z Tue, 25 Feb 2025 22:35:19 GMT:read config 
2025-02-25T22:35:22.0699082Z  &{Repo:ericcornelissen/eslint-plugin-top CorrelationId:1902866b-96f6-411a-a3a7-3e6e8cd9c727 RunId:13532288617 WorkingDirectory:/home/runner/work/eslint-plugin-top/eslint-plugin-top APIURL:https://agent.api.stepsecurity.io/v1 OneTimeKey: Endpoints:map[github.com.:[{domainName:github.com. port:443}] kv4gacprodeus2file3.blob.core.windows.net.:[{domainName:kv4gacprodeus2file3.blob.core.windows.net. port:443}]] EgressPolicy:block DisableTelemetry:false DisableSudo:true DisableFileMonitoring:false Private:false}
2025-02-25T22:35:22.0701689Z 
2025-02-25T22:35:22.0702029Z Tue, 25 Feb 2025 22:35:19 GMT:Step Security Job Correlation ID: 1902866b-96f6-411a-a3a7-3e6e8cd9c727
2025-02-25T22:35:22.0702821Z 
2025-02-25T22:35:22.0702994Z Tue, 25 Feb 2025 22:35:19 GMT:started process monitor
2025-02-25T22:35:22.0703349Z 
2025-02-25T22:35:22.0703700Z Tue, 25 Feb 2025 22:35:19 GMT:Monitor Processes called
2025-02-25T22:35:22.0704248Z Tue, 25 Feb 2025 22:35:19 GMT:NewAuditClient created
2025-02-25T22:35:22.0704885Z Tue, 25 Feb 2025 22:35:19 GMT:Status is enabled
2025-02-25T22:35:22.0705441Z Tue, 25 Feb 2025 22:35:19 GMT:Rules deleted
2025-02-25T22:35:22.0706594Z Tue, 25 Feb 2025 22:35:19 GMT:File monitor added for /home/runner/work/eslint-plugin-top/eslint-plugin-top
2025-02-25T22:35:22.0707619Z Tue, 25 Feb 2025 22:35:19 GMT:Agent file monitor added
2025-02-25T22:35:22.0708247Z Tue, 25 Feb 2025 22:35:19 GMT:Docker's daemon.json file monitor added
2025-02-25T22:35:22.0708934Z Tue, 25 Feb 2025 22:35:19 GMT:Systemd's resolved.conf file monitor added
2025-02-25T22:35:22.0709510Z Tue, 25 Feb 2025 22:35:19 GMT:Net monitor added
2025-02-25T22:35:22.0710028Z Tue, 25 Feb 2025 22:35:19 GMT:Process monitor added
2025-02-25T22:35:22.0710528Z Tue, 25 Feb 2025 22:35:19 GMT:receive called
2025-02-25T22:35:22.0710843Z 
2025-02-25T22:35:22.0711315Z Tue, 25 Feb 2025 22:35:19 GMT:domain resolved: agent.api.stepsecurity.io., ip address: 52.88.221.136, TTL: 60
2025-02-25T22:35:22.0712519Z Tue, 25 Feb 2025 22:35:19 GMT:domain resolved: kv4gacprodeus2file3.blob.core.windows.net., ip address: 20.60.63.97, TTL: 37
2025-02-25T22:35:22.0713628Z Tue, 25 Feb 2025 22:35:19 GMT:domain resolved: github.com., ip address: 140.82.112.4, TTL: 30
2025-02-25T22:35:22.0714546Z Tue, 25 Feb 2025 22:35:19 GMT:domain resolved: codeload.github.com., ip address: 140.82.114.10, TTL: 60
2025-02-25T22:35:22.0715788Z Tue, 25 Feb 2025 22:35:19 GMT:domain resolved: actions-results-receiver-production.githubapp.com., ip address: 140.82.114.23, TTL: 57
2025-02-25T22:35:22.0716832Z 
2025-02-25T22:35:22.0717018Z Tue, 25 Feb 2025 22:35:20 GMT:updated resolved
2025-02-25T22:35:22.0717359Z 
2025-02-25T22:35:22.0717549Z Tue, 25 Feb 2025 22:35:20 GMT:set docker config
2025-02-25T22:35:22.0717864Z 
2025-02-25T22:35:22.0717870Z 
2025-02-25T22:35:22.0718719Z Tue, 25 Feb 2025 22:35:20 GMT:Allowed domains:map[github.com.:[{github.com. 443}] kv4gacprodeus2file3.blob.core.windows.net.:[{kv4gacprodeus2file3.blob.core.windows.net. 443}]]
2025-02-25T22:35:22.0719691Z 
2025-02-25T22:35:22.0719861Z Tue, 25 Feb 2025 22:35:20 GMT:disabled sudo
2025-02-25T22:35:22.0720289Z Tue, 25 Feb 2025 22:35:20 GMT:done
2025-02-25T22:35:22.0720548Z 
2025-02-25T22:35:22.0720730Z Tue, 25 Feb 2025 22:35:21 GMT:post_event called
2025-02-25T22:35:22.0721054Z 
2025-02-25T22:35:22.0721171Z status:
2025-02-25T22:35:22.0721453Z Initialized
2025-02-25T22:35:22.0781132Z Cleaning up orphan processes
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,8 @@ RUN git checkout c5157ee9f560a93364708ff2276dba71d1967cc3
 
 # Apply improvement patches to ODGen
 RUN git apply ../patches/exit_code.patch \
-	&& git apply ../patches/pin_babel_deps.patch
+	&& git apply ../patches/pin_babel_deps.patch \
+	&& git apply ../patches/pip_requirements.patch
 
 # Install ODGen dependencies
 RUN apt-get update \

--- a/patches/pip_requirements.patch
+++ b/patches/pip_requirements.patch
@@ -1,0 +1,11 @@
+diff --git a/requirements.txt b/requirements.txt
+index 48e617d..991d2e3 100644
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -2,6 +2,5 @@ func_timeout
+ networkx
+ sty==1.0.0rc0
+ tqdm
+-pysimplegui
+ flask
+ tinydb

